### PR TITLE
Allow changing device hostname.

### PR DESCRIPTION
**Pull request by robputt**
- Sets default hostname at first boot to "nebra-<modified_eth_mac>.local" e.g. "nebra-5F1CAB.local".
- Displays hostname on the diagnostics page.
- Allows user to update hostname on the device configuration page.

### CODE REVIEW

File: hw_diag/templates/diagnostics_page_light_miner.html [hunk 1]
```diff
@@ -63,9 +63,9 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                   {% endif %}
                 </tr>
                 <tr>
-                  <td>Hostname</td>
+                  <td>Hostname (IP Address)</td>
```
Comment (shawaj) [inline hw_diag/templates/diagnostics_page_light_miner.html:66]: Minor comment but maybe Local IP would be better?

I guess we could also put the Public IP on diagnostics as well but probably better on a different line?

Comment (robputt) [reply hw_diag/templates/diagnostics_page_light_miner.html:66]: I think we leave it like this for now, and then we can make it look pretty with the UI updates and include both local and external IPs...

Comment (shawaj) [reply hw_diag/templates/diagnostics_page_light_miner.html:66]: Sounds like a good plan 👌

**Review by shawaj — APPROVED** [review]
LGTM

**Pull request merged**